### PR TITLE
Dwv even

### PR DESCRIPTION
**Pull request by danwventura**

### CODE REVIEW

**Comment by danwventura** [comment]
@BlaiseGratton

File: Mathmagician.Tests/EvenTest.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    {
+        
+
+        [TestMethod]
+        public void GiveBackEvensList() //Checking that we are returning a list
+        {
+            // Arrange section
+            Even myEven = new Even();
+            // Act
+            var evensTestList = myEven.CreateEvenSequence(5);
```
Comment (madduxTim) [inline Mathmagician.Tests/EvenTest.cs:18]: Jurnell requested that change the vars here to List with the list type.

File: Mathmagician/Even.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mathmagician
+{
+    public class Even
+    {
+        public List<int> CreateEvenSequence(int user_integer)
+        {
+            var evensList = new List<int>();
```
Comment (madduxTim) [inline Mathmagician/Even.cs]: Change var to List<int>

**Comment by madduxTim** [comment]
@BlaiseGratton

**Comment by BlaiseGratton** [comment]
👍

**Comment by madduxTim** [comment]
👍

**Pull request merged**